less/variables.less

```diff
@@ -1 +1,25 @@
-// Variables
\ No newline at end of file
+// Variables
+@mobile: ~"( max-width: 500px )";
+
+// Navigation background: #F6F6F6
+@nav-background: #F6F6F6;
+// Navigation font color: #454A59
+@nav-font-color: #454A59;
+// Navigation border color: #F1F1F1;
+@nav-border-color:#F1F1F1;
```

Comment:
Great job using variable names that are descriptive! This makes understanding your code so much easier when you come back to it later, or more importantly, when someone else looks at/works on your code. Really awesome to see you with this good habit early.